azurecontainerapps.ts

```diff
@@ -608,8 +608,9 @@ export class azurecontainerapps {
         } else if (this.shouldCreateOrUpdateContainerAppWithUp) {
             await this.appHelper.createOrUpdateContainerAppWithUp(this.containerAppName, this.resourceGroup, this.commandLineArgs);
         } else {
-            // Update the Container App using the 'up' command
-            await this.appHelper.updateContainerAppWithUp(this.containerAppName, this.resourceGroup, this.commandLineArgs, this.ingress, this.targetPort);
+            // Update the Container App using the 'containerapp update' and 'ingress update' commands
+            await this.appHelper.updateContainerApp(this.containerAppName, this.resourceGroup, this.commandLineArgs)
```

Comment:
Synced offline, since we don't use the `up` command to update the container app anymore, we should change the name of the global variable `this.shouldUseUpdateCommand` to clearly indicate when the `update ingress` command needs to be executed.

Reply:
sounds good! updated in the latest commit

Reply:
@snehapar9 -- I'm not sure I understand your comment _since we don't use the `up` command to update the container app anymore_ because I do see a function call a few lines up `updateContainerAppWithUp` that still uses `up`. Would you mind elaborating a bit more?

Reply:
@cormacpayne This comment does not need to be addressed anymore.
@harryli0108's initial changes had `up` removed and made calls to only `update`. But since `update` does not support registry credentials, `up` was added back.